Tests/SpriteKitTests/SKNodeExtensionTests.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  SpriteKitTests.swift
+//  SwifterSwift
+//
+//  Created by Olivia Brown on 5/28/18.
+//  Copyright © 2018 SwifterSwift
+//
+
+import XCTest
+@testable import SwifterSwift
+
+#if canImport(SpriteKit)
+import SpriteKit
+
+@testable import SwifterSwift
+
+final class SKNodeExtensionTests: XCTestCase {
+
+    func testDescendants() {
+        let scene = SKScene()
+        let childOne = SKNode()
+        scene.addChild(childOne)
+        let childTwo = SKNode()
+        childOne.addChild(childTwo)
+        XCTAssertEqual(scene.descendants(), [childOne, childTwo])
+        XCTAssertEqual(childOne.descendants(), [childTwo])
+        XCTAssertEqual(childTwo.descendants(), [])
+    }
+
+    func testSKNodeCenter() {
+        let scene = SKScene(size: .init(width: 100, height: 100))
```

Comment:
The standard in this project (although I'm sure it's not used in a few places) is to use the explicit initialiser.
```suggestion
        let scene = SKScene(size: CGSize(width: 100, height: 100))
```

Reply:
I've added review changes and squashed all into a single commit for convenience. 🚀 
Please re-review @LucianoPAlmeida @guykogus